src/App.js

```diff
@@ -10,15 +10,37 @@ import { useAuthState } from 'react-firebase-hooks/auth';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 firebase.initializeApp({
-  apiKey: "AIzaSyD0WUxAE05_qqa9rOCGYP5USp_IkZ8T4YQ",
-  authDomain: "react-firebase-chat-888c1.firebaseapp.com",
-  projectId: "react-firebase-chat-888c1",
-  storageBucket: "react-firebase-chat-888c1.appspot.com",
-  messagingSenderId: "878837492403",
-  appId: "1:878837492403:web:c75838294caa341c13dec6",
-  measurementId: "G-2930JCZS81"
+  apiKey: "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
+  authDomain: "chat-app-15946.firebaseapp.com",
+  projectId: "chat-app-15946",
+  storageBucket: "chat-app-15946.appspot.com",
+  messagingSenderId: "533178884935",
+  appId: "1:533178884935:web:ef9483d776d0107ffe6262",
+  measurementId: "G-P58VBCQ0SY"
```

Comment:
not able to send message, kindly change the configuration details again back to my database